spring-kotlin/aws-ssh-server-manager/src/main/kotlin/dev/santos/awssshservermanager/lib/aws/config/AwsSsmConfig.kt

```diff
@@ -0,0 +1,44 @@
+package dev.santos.awssshservermanager.lib.aws.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.ssm.SsmClient
+import java.net.URI
+
+@Configuration
+class AwsSsmConfig {
+    @Value("\${aws.ssm.region}")
+    private val regionStr: String? = null
+
+    @Value("\${aws.ssm.endpointUri}")
+    private val endpointUriStr: String? = null
```

Comment:
Is it normal in kotlin to `Str` at the end of the string? (The fact that their type is "string" should give that up)

Reply:
That is just me... because that endpointUri will be an URI later on and giving it the same name might be misleading
I actually don't need to assign it to a variable, so I can be less verbose as you suggest

I don't like to rely on scope to understand what the variable type is